extensions/typescript-language-features/src/extension.browser.ts

```diff
@@ -98,11 +146,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<Api> {
 	context.subscriptions.push(lazilyActivateClient(lazyClientHost, pluginManager, activeJsTsEditorTracker, async () => {
 		await preload(logger);
 	}));
+	context.subscriptions.push(vscode.workspace.registerFileSystemProvider('vscode-global-typings', new MemFS(), {
+		isCaseSensitive: true,
+		isReadonly: false
```

Comment:
As I understand it, this tells VS Code that the FileSystemProvider is readonly: aka, that no write operations are supported.

Since you made this comment, things and changed, and neither of them are readonly nice.